crates/runtime/src/http/v1/mod.rs

```diff
@@ -93,30 +94,50 @@ impl utoipa::IntoParams for Format {
 #[derive(Default, Debug, Serialize, Deserialize)]
 #[serde(rename_all = "lowercase")]
 /// The various formats that the Arrow data can be converted and returned from HTTP requests.
-pub enum ArrowFormat {
+pub enum DataFormat {
```

Comment:
This should really be `ResponseFormat`. DataFormat is part of the response.

Reply:
Or technically, `ResponseMimeType`

Reply:
Updated, thank you